javascript/javascript1/week4/VoiceAssistant.js

```diff
@@ -0,0 +1,180 @@
+const commandHistory = []
+let curUser = ""
```

Comment:
Make sure when you name variables to use full words. This will make your code faster to read:
`let currentUserName` or `let userName`

Reply:
Here, I am comparing the old names with the current value. If I have not added to commandHistory , how can I check the currentName already exists?

Reply:
Ah ok. So you want a list of all the past names?
I thought you only ever wanted to keep one.

Reply:
Yes. I want the list of old names😀.